app/sagas/selectServer.ts

```diff
@@ -130,30 +194,27 @@ const handleSelectServer = function* handleSelectServer({ server, version, fetch
 		setRoles();
 		setEnterpriseModules();
 
-		let serverInfo;
-		if (fetchVersion) {
-			serverInfo = yield getServerInfo({ server, raiseError: false });
-		}
-
 		// Return server version even when offline
-		const serverVersion = (serverInfo && serverInfo.version) || version;
+		const serverVersion = (serverInfo && serverInfo.version) || (version as string);
 
 		// we'll set serverVersion as metadata for bugsnag
 		logServerVersion(serverVersion);
-		yield put(selectServerSuccess(server, serverVersion));
+		yield put(selectServerSuccess({ server, version: serverVersion, name: serverInfo?.name || 'Rocket.Chat' }));
```

Comment:
Is this Rocket.Chat really fixed?

Reply:
Also a ts thing. It can be undefined, but it shouldn't.
In case it is, we fallback to that.